pkg/pillar/cmd/volumemgr/handledownloader.go

```diff
@@ -46,7 +46,7 @@ func AddOrRefcountDownloaderConfig(ctx *volumemgrContext, blob types.BlobStatus)
 	// where should the final downloaded file be?
 	// Pick a name based on existing info about object to persist it across reboots
 	idHash := sha256.New()
-	idHash.Write(blob.DatastoreID.Bytes())
+	idHash.Write(blob.DatastoreIDsList[0].Bytes())
```

Comment:
Looks like this should loop over all the IDs in the list and add each one to the hash to make sure the resulting string changes if the list of IDs changes.

Reply:
Yes, makes sense. Thanks.